external_drivers/dhi_dfs/mdal_dhi_dfs.hpp

```diff
@@ -0,0 +1,369 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2020 Vincent Cloarec (vcloarec at gmail dot com)
+*/
+
+#ifndef MDAL_DHI_DFS_HPP
+#define MDAL_DHI_DFS_HPP
+
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+#include <memory>
+#include <unordered_map>
+#include <assert.h>
+
+#include "eum.h"
+#include "dfsio.h"
+
+#undef min
+#undef max
```

Comment:
hmm, is this required?

Reply:
yes, without that no build with Windows.